contents/tutorials/session-metrics.md

```diff
@@ -6,67 +6,102 @@ sidebar: Docs
+> **Tip:** Be sure to set the Y-axis unit in your chart back to `None`.
 
-In many ways, average session duration is more useful than the sum or total time spent on site. A large number of sessions can increase total time on site, which is driven by brand, marketing, and distribution. Average session duration is more driven by the quality and content of the pages and application, which engineers and product managers have more control over. 
+<ProductScreenshot
+    imageLight = "https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_11_26_at_11_35_08_2x_20bea3290f.png"
+    imageDark = "https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_11_26_at_11_35_47_2x_8f91b16f3c.png"
+    alt="Pages per session" 
+    classes="rounded"
+/>
 
-Once you calculate average session duration, you can filter the series to get information based on source or referring domain, device type, active feature flags, cohorts, and more. All of these filters help you get closer to the actual user behavior you are trying to understand.
+You can filter these series to provide details on sections and funnels on your site. Clicking on the charts will provide a list of the users included in that data. If there is a [session replay](/docs/session-replay) available for that user session, there will be a link for you to go watch it.
 
-## Pages per session
+## Analyzing sessions with SQL
 
-Another useful average to visualize is the average number of pages per session. To do this set up two graph series, one for total pageview count and a second for pageview unique sessions. You can then use the `A/B` formula to calculate the number of pages per session. 
+If trends don't give you the details you need, you can access the raw session data with SQL. It is located in the `sessions` table.
 
-> **Tip:** Be sure to set the Y-axis unit in your chart back to `None`.
+For example, to get the average number of unique URLs per session, you can create a new SQL insight and run the following query:
 
-![Pages per session](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/tutorials/session-metrics/pages-per-session.png)
+```sql
+select avg($num_uniq_urls) from sessions
+```
 
-Again, you can filter these series to better identify the types of users you want, or to provide details on sections and funnels on our site. Clicking on the charts will provide a list of the users included in that data. If there is a [session recording](/manual/recordings) available for that user session, there will be a link for you to go directly watch it.
+### Connecting sessions to other data
 
-### Understanding how users spend their time
+A big use case for this is combining session data with other event data. Because all events in a `$session_id` property, we can use a `JOIN` to combine session data with event data.
```

Comment:
```suggestion
A big use case for this is combining session data with other event data. Because events have `$session_id` property, we can use a `JOIN` to combine session data with event data.
```